crud/src/test/java/com/redhat/lightblue/crud/rdbms/RDBMSCRUDControllerTest.java

```diff
@@ -0,0 +1,171 @@
+        com.redhat.lightblue.metadata.rdbms.model.Operation delete = new com.redhat.lightblue.metadata.rdbms.model.Operation();
+        delete.setName(LightblueOperators.DELETE);
+        ArrayList<Expression> expressionList = new ArrayList<>();
+        Statement e = new Statement();
+        e.setSQL("SQL");
+        e.setType("delete");
+        expressionList.add(e);
+        delete.setExpressionList(expressionList);
+        rdbms.setDelete(delete);
+        rdbms.setDialect(DialectOperators.ORACLE);
+
+        //rdbms details for this test
+        SQLMapping sQLMapping = new SQLMapping();
+        ArrayList<Join> joins = new ArrayList<>();
+        Join e1 = new Join();
+        ArrayList<ProjectionMapping> projectionMappings = new ArrayList<>();
+        ProjectionMapping e2 = new ProjectionMapping();
+        e2.setColumn("col");
+        e2.setField("fie");
+        projectionMappings.add(e2);
+        e1.setProjectionMappings(projectionMappings);
+        ArrayList<Table> tables = new ArrayList<>();
+        Table e3 = new Table();
+        e3.setName("table");
+        tables.add(e3);
+        e1.setTables(tables);
+        joins.add(e1);
+        sQLMapping.setJoins(joins);
+        ArrayList<ColumnToField> columnToFieldMap = new ArrayList<>();
+        ColumnToField e4 = new ColumnToField();
+        columnToFieldMap.add(e4);
+        sQLMapping.setColumnToFieldMap(columnToFieldMap);
+        rdbms.setSQLMapping(sQLMapping);
+        Bindings bindings = new Bindings();
+        ArrayList<InOut> inList = new ArrayList<>();
+        InOut e5 = new InOut();
+        e5.setColumn("col");
+        e5.setField(new Path("fie"));
+        inList.add(e5);
+        bindings.setInList(inList);
+        delete.setBindings(bindings);
+
+        //System.out.println(String.valueOf(rdbms));
+    }
+
+    public static Projection p = new Projection() {
+        @Override
+        public JsonNode toJson() {
+            return null;
+        }
+    };
+
+    @Test
+    public void testInsert() throws Exception {
+        cut.insert(c, p);
+        assertNotNull(c.getErrors());
+        assertTrue(c.getErrors().isEmpty());
+    }
+
+
+    @Test
+    public void testDelete() throws Exception {
+        FieldComparisonExpression fieldComparisonExpression = new FieldComparisonExpression(new Path("fie"), BinaryComparisonOperator._eq, new Path("fie"));
+        cut.delete(c, fieldComparisonExpression);
+        assertNotNull(c.getErrors());
+        assertTrue(c.getErrors().isEmpty());
+    }
+
+    @Test
+    public void testFind() throws Exception {
+        FieldComparisonExpression fieldComparisonExpression = new FieldComparisonExpression(new Path("fie"), BinaryComparisonOperator._eq, new Path("fie"));
+        cut.find(c, fieldComparisonExpression,new FieldProjection(new Path("fie"),true,false), null, null, null);
+        assertNotNull(c.getErrors());
```

Comment:
Yeah,  there is space for much more tests, those were just some simple basic ones which can be used as based to create them. This PR was not aiming to test much further as I made it but didn't have enough time to spend on this, but it helped to fix an initialization problem and also it can be the base to we create more tests (as I mentioned earlier).